src/server/pegasus_server_impl.cpp

```diff
@@ -1360,6 +1398,7 @@ ::dsn::error_code pegasus_server_impl::start(int argc, char **argv)
     if (db_exist) {
         _last_committed_decree = _meta_store->get_last_flushed_decree();
         _pegasus_data_version = _meta_store->get_data_version();
+        _usage_scenario = _meta_store->get_usage_scenario();
```

Comment:
I think it will cause core while rolling_update server. The table created before won't have `usage_scenario` in meta column family, while rolling update, the `db_exist` is true, it will assert on Line72 in function `get_usage_scenario`

Reply:
Should not, she `set_usage_scenario`  at first open
https://github.com/apache/incubator-pegasus/blob/ec3b23092e4c677ac14a2ab0e7b0c3966dec5a15/src/server/pegasus_server_impl.cpp#L1417

Reply:
Indeed, maybe `get_usage_scenario()` should return `ROCKSDB_ENV_USAGE_SCENARIO_NORMAL` if we couldn't find usage_scenario in rocksdb meta column family.